### ISSUE TITLE
Event node shows all devices as disabled

### ISSUE THREAD

**Issue by oldironUK** [Samsung/SamsungAutomationStudio#49]
Status node works successfully however event node shows all my devices as disabled.

Any suggestions?

Thanks

**Comment by iamjaehun** [comment]
Devices loaded by PAT from the my device node cannot be used in the event node, but can use the capabilities selected in the device profile node. Event nodes are used to build smart apps. For more information on how to build smart apps, check out our tutorial page.
https://developer.samsung.com/automation/v2-smartthings-automation.html

**Comment by oldironUK** [comment]
Hi
Thanks for quick reply - I followed your link above - can you give me a pointer to be able to add the automation node? :
"Add a node for the SmartThings Automation (webhook)."

I can see this at the moment in the automation node:
http://192.168.1.148:1880

What would I need to add to the above that to be able to capture a button being pressed

Samsung  Automation Studio is showing this input for the command node:

https://as-638e3717-ff99-4e21-9049-3b9cd85b3174.app.scf.use2.devground.io/3ed7e864.94df48/button

I'm sure this is a really basic question - but all I want to do is capture a button press, which of course is so easy in Webcore!

Thanks in advance

**Comment by iamjaehun** [comment]
The automation node also shows an endpoint which is a SmartApp webhook endpoint.
You need to register the endpoint on the SmartThings Developer Workspace.
This requires a public endpoint with https.
(Information on SmartApp registration can be found [here](https://developer-preview.smartthings.com/docs/connected-services/app-registration/).)

But your node-red(http://192.168.1.148:1880) seems to have only private address. 
This means that the address can only be accessed from the your internal network. (Perhaps only the devices connected to your router.)
Make sure node-red has a public address with https certificate applied so that it can be accessed externally.

I know this is not a basic course.
So we've launched a new node called _My Device SSE_ in Samsung Automation Studio to make it easier to listen to events.
The _My Device SSE_ node can receive events from devices retrieved by the PAT from the _My Device_ node.
Go to Samsung Automation Studio and checkout the _My Device SSE_ node.

**Comment by oldironUK** [comment]
Wow - refreshed my screen and there is the new _My Device SSE_ node.   Thanks very much!

Worked straight out of the box - I probably haven't deployed it is the best way but here it is in a working state for anyone else who might be looking

Is this node also going to be added to the _"node-red-contrib-samsung-automation-studio-nodes 1.1.12"_?



![image](https://user-images.githubusercontent.com/57288218/145900802-fcc126dc-0dae-4c8e-94d1-cc8bdeb6a70f.png)

**Comment by iamjaehun** [comment]
Yes, we are planning to release the _My Device SSE_ node to the _node-red-contrib-samsung-automation-studio-nodes_ as well. The release date has not been set yet, but we will try to release it as soon as possible.

**Comment by oldironUK** [comment]
Thanks for your help - I look forward to see the new node in _node-red-contrib-samsung-automation-studio-nodes_!

**Comment by oldironUK** [comment]
> Yes, we are planning to release the _My Device SSE_ node to the _node-red-contrib-samsung-automation-studio-nodes_ as well. The release date has not been set yet, but we will try to release it as soon as possible.

Any update on the due date for this - "_My Device SSE_ node to the _node-red-contrib-samsung-automation-studio-nodes_"

**Comment by oldironUK** [comment]
> Yes, we are planning to release the _My Device SSE_ node to the _node-red-contrib-samsung-automation-studio-nodes_ as well. The release date has not been set yet, but we will try to release it as soon as possible.

Any update on the due date for this - "My Device SSE node to the node-red-contrib-samsung-automation-studio-nodes"

**Comment by danmiles86** [comment]
@iamjaehun also looking for an update on this please! :)

**Comment by dangruhn** [comment]
It is now almost 2 years, any update?
